src/app/dashboard/_dashboard.scss

```diff
@@ -7,35 +8,114 @@ $paperCardBackgroundColor: #212121 !default;
 
 #dashboardContainer {
 	background-color: $dashboardContainerBackgroundColor;
+	height: 100%;
 
-	.userbar {
+	// typographic defaults
+	font-family: Roboto, sans-serif;
+	font-style: normal;
+	font-size: 1em;
+	line-height: 1.2em;
+
+	header {
 		position: fixed;
 		top: 0;
-		left: 0;
+		display: block;
 		width: 100%;
-		height: 46px;
 		background-color: $userbarBackgroundColor;
-		padding: 5px;
-		margin-bottom: 10px;
+		text-color: $userbarTextColor;
+		box-shadow: rgba(0, 0, 0, 0.25) 0px 2px 5px 0px;
+		margin: 0;
+
+			nav {
+			top: 0;
+			left: 0;
+			padding: 0;
+
+				ul {
+				list-style-type: none;
+				display: block;
+				float: right;
+				margin: 0;
+				padding: 0;
+
+				li {
+					display: block;
+					position: relative;
+					
+					a {
+						display: block;
+						margin: 0.25rem;
+						padding: 0.65rem 1.25rem;
+						text-color: $userbarTextColor;
+						text-decoration: none;
+
+						&:link, &:visited {
+							color: $userbarTextColor;
+						}
+						&:hover {
+							background-color: rgba(255,255,255,0.2);
+						}
+					}
+				}
+			}
+		}
 	}
 
-	.paperCard {
+	#tray {
 		display: block;
-		width: 240px;
-		height: 112px;
-		margin: 24px;
-		color: $paperCardBackgroundColor;
-		font-family: Roboto sans, sans-serif ;
-		margin: 10px ;
-		border-radius: 2px;
-
-		h1 {
-			font-weight: bold;
-			padding: 8px;
-			text-align: left;
-		}
-		p {
-			padding: 8px;
+		padding-top: 4rem;
+
+		.paperCard {
+			display: inline-block;
+			width: 16rem;
+			margin: 0.5rem;
+			padding: 0;
+			color: $paperCardBackgroundColor;
+			font-family: Roboto sans, sans-serif ;
```

Comment:
i see you are using the Roboto sans multiple times.
lets go ahead and create a variable for this called `$mainFont`